RELEASE/scripts/autoscend/quests/level_12.ash

```diff
@@ -1859,7 +1759,7 @@ boolean L12_themtharHills()
 		use_familiar(to_familiar(get_property("auto_familiarChoice")));
 	}
 	equipMaximizedGear();
-	float meatDropHave = meat_drop_modifier();
+	float meatDropHave = provideMeat(400, false, true);
```

Comment:
why 400? add comment explaining reasoning

Reply:
Line 1749 says that the target is 400 minimum. Changing it to be the 1800 line like below just with doEverything false but keep speculative as true too.